app/providers/services/user/index.js

```diff
@@ -64,6 +65,17 @@ class UserService {
     return updatedUser
   }
 
+  async addAvatarUrl(users) {
+    const avatarUrlPromises = users.map(async (user) => {
+      if (user.avatar_object) {
+        user["avatar_url"] = await this.storageService.downloadFile(user._id, user.avatar_object.file_id)
```

Comment:
This looks good.

The only thing is let's rename `downloadFile` to `getFileDownloadUrl`